From eaadb1446c7c0e9cf183e30400ff048b913da495 Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Tue, 14 Sep 2010 17:46:02 +0800
Subject: [PATCH] ENGR00131475 imx5 MSL-CLK: set correct clock lookup item for csi and di

The clk_get function in clkdev should fit exact dev_id name,
set dev_id name then call clk_get() with dev parameter=NULL
would not get the correct clk handler.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 arch/arm/mach-mx5/clock.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/mach-mx5/clock.c b/arch/arm/mach-mx5/clock.c
index b29f4c0..7673dc9 100644
--- a/arch/arm/mach-mx5/clock.c
+++ b/arch/arm/mach-mx5/clock.c
@@ -4104,10 +4104,10 @@ static struct clk_lookup lookups[] = {
 	_REGISTER_CLOCK("mxc_sdma", "sdma_ahb_clk", sdma_clk[0]),
 	_REGISTER_CLOCK("mxc_sdma", "sdma_ipg_clk", sdma_clk[1]),
 	_REGISTER_CLOCK(NULL, "ipu_clk", ipu_clk[0]),
-	_REGISTER_CLOCK("mxc_ipu", "ipu_di0_clk", ipu_di_clk[0]),
-	_REGISTER_CLOCK("mxc_ipu", "ipu_di1_clk", ipu_di_clk[1]),
-	_REGISTER_CLOCK("mxc_ipu", "csi_mclk1", csi0_clk),
-	_REGISTER_CLOCK("mxc_ipu", "csi_mclk2", csi1_clk),
+	_REGISTER_CLOCK(NULL, "ipu_di0_clk", ipu_di_clk[0]),
+	_REGISTER_CLOCK(NULL, "ipu_di1_clk", ipu_di_clk[1]),
+	_REGISTER_CLOCK(NULL, "csi_mclk1", csi0_clk),
+	_REGISTER_CLOCK(NULL, "csi_mclk2", csi1_clk),
 	_REGISTER_CLOCK("tve.0", NULL, tve_clk),
 	_REGISTER_CLOCK("mxcintuart.0", NULL, uart1_clk[0]),
 	_REGISTER_CLOCK("mxcintuart.1", NULL, uart2_clk[0]),
-- 
1.5.4.4

